converters/toZigbee.js

```diff
@@ -240,14 +240,26 @@ const converters = {
         },
     },
     ballast_config: {
-        key: ['ballast_config'],
+        key: ['ballast_config',
+            'ballast_physical_minimum_level',
+            'ballast_physical_maximum_level',
+            'ballast_minimum_level',
+            'ballast_maximum_level'],
         // zcl attribute names are camel case, but we want to use snake case in the outside communication
         convertSet: async (entity, key, value, meta) => {
-            value = utils.toCamelCase(value);
-            for (const [attrName, attrValue] of Object.entries(value)) {
-                const attributes = {};
-                attributes[attrName] = attrValue;
-                await entity.write('lightingBallastCfg', attributes);
+            if (key === 'mode_phase_control') {
```

Comment:
Shouldn't `mode_phase_control` also be in the key array above?

Reply:
Good catch! Was mixing things up, that should have been `ballast_config`...